NAMESPACE

```diff
@@ -17,6 +17,7 @@ export(is.mortaar_life_table_list)
 export(life.table)
 export(prep.life.table)
 importFrom(Rdpack,reprompt)
+importFrom(demography,cm.spline)
```

Comment:
Probably not necessary to add another dependency: cm.spline is only a wrapper for stats::spline, see below